tests/BenchmarkDotNet.Tests/Attributes/VerifiedFiles/ParamsAllValuesVerifyTests.BenchmarkShouldProduceSummary_WithNotAllowedFlagsEnumError.verified.txt

```diff
@@ -7,9 +7,9 @@ Frequency: 2531248 Hz, Resolution: 395.0620 ns, Timer: TSC
   DefaultJob : extra output line
 
 
-    Method | ParamProperty |     Mean |   Error |  StdDev |
----------- |-------------- |---------:|--------:|--------:|
- Benchmark |             0 | 102.0 ns | 6.09 ns | 1.58 ns |
+Method    | ParamProperty | Mean     | Error   | StdDev  | 
+--------- |-------------- |---------:|--------:|--------:|
+Benchmark | 0             | 102.0 ns | 6.09 ns | 1.58 ns | 
```

Comment:
Can those extra spaces on the end be removed?

Reply:
> Can those extra spaces on the end be removed?

I should add some logic to TrimEnd() the "TableHeaderSeparator" and TableColumnSeparator" when we want to print them for last column. I wasn't sure that's what we want but if its okay with you I can add it.